pom.xml

```diff
@@ -584,6 +589,12 @@
         <groupId>com.google.api</groupId>
         <artifactId>gax-grpc</artifactId>
         <version>${gax-grpc.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.google.api.grpc</groupId>
+            <artifactId>proto-google-common-protos</artifactId>
```

Comment:
Why does this need to be excluded here? We have control over the gax-grpc -> proto-google-common-protos dependency and we can fix it up if there's a problem...